### ISSUE TITLE
 Небольшой дожд…

### ISSUE THREAD

**Issue by romanesko** [chubin/wttr.in#81]
Видимо где-то закралась ошибка с > и >=
(если там, конечно, было не слово «дождик») )

**Comment by chubin** [comment]
Там может быть например "Небольшой дождь со снегом" или ещё что-нибудь.
Какой это был город?

**Comment by romanesko** [comment]
Москва на 3 марта

![Alt text](https://monosnap.com/file/bQ4xSu3IjB3w3w9fjbOOUV6q8HpPZE.png)

**Comment by chubin** [comment]
Да, там как раз "Небольшой дождь со снегом". Видно по картинке + я проверил, какие там данные реально стоят. Но я заметил другую ошибку: на английском и немецком указан град (sleet на английском, Graupel на немецком, grésil на французском)

**Comment by alex-liberty** [comment]
А тут? ![2017-04-05](https://cloud.githubusercontent.com/assets/12385901/24723304/be4c47ec-1a4f-11e7-90fd-25a4aa3d39ef.png)
Локация Минск

**Comment by chubin** [comment]
Это всё же баг.
Спасибо @alex-liberty и @romanesko 
Надо фиксить
